### PR TITLE
Add EricVideos xPath scene, performer scraper

### PR DESCRIPTION
**Pull request by DogmaDragon**
Requested here https://github.com/stashapp/CommunityScrapers/issues/73#issuecomment-1092940217

Grabs a poster from the gallery instead of the player.

### CODE REVIEW

File: scrapers/EricVideos.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      - ericvideos.com
+    scraper: performerScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1/text()
+      Details:
+        selector: //div[@class="texte"]/text()
+      Tags:
+        Name: //a[@class='badge badge-dark']/text()
```
Comment (bnkai) [inline scrapers/EricVideos.yml:19]:
```yaml
        Name:
          selector: //a/div[@class='acteur']/div[@class='nom']/text()
        URL:
          selector:  //a[div[@class='acteur']]/@href
          postProcess:
            - replace:
                - regex: ^
                  with: https://www.ericvideos.com
```
the URL can also be scraped

File: scrapers/EricVideos.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          selector: //a/div[@class='acteur']/div[@class='nom']/text()
+      Studio:
+        Name:
+          fixed: EricVideos
+      Image:
+        selector: //div[@class='preview'][1]/a/img[@class='preview']/@src
+        postProcess:
+          - replace:
+              - regex: (.*)
+                with: https://www.ericvideos.com/$1
```
Comment (bnkai) [inline scrapers/EricVideos.yml:31]: simpler
```yaml
                - regex: ^
                  with: https://www.ericvideos.com
```

File: scrapers/EricVideos.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        postProcess:
+          - feetToCm: true
+      Gender:
+        fixed: male
+      Image:
+        selector: //div[@class='photo_acteur']/img[@class='thumbnail']/@src
+        postProcess:
+          - replace:
+              - regex: (.*)
+                with: https://www.ericvideos.com/$1
```
Comment (bnkai) [inline scrapers/EricVideos.yml:48]:
```yaml
                - regex: ^
                  with: https://www.ericvideos.com/
```

**Comment by DogmaDragon** [comment]
Verified everything works.

**Pull request merged**